django/db/models/base.py

```diff
@@ -538,6 +542,18 @@ def __hash__(self):
             raise TypeError("Model instances without primary key value are unhashable")
         return hash(self.pk)
 
+    def __copy__(self):
```

Comment:
This approach is based on the implementation of Field.__copy__ and used for the same reason.